cmd/fossa/main.go

```diff
@@ -49,13 +47,23 @@ func main() {
 		cli.StringFlag{Name: "m, modules", Usage: "the modules to build and analyze"},
 		cli.BoolFlag{Name: "o, output", Usage: analyzeOutputUsage},
 		cli.BoolFlag{Name: "allow-unresolved", Usage: analyzeAllowResolvedUsage},
-		cli.BoolFlag{Name: "no-upload", Usage: analyzeNoUploadUsage},
 		cli.BoolFlag{Name: "b, build", Usage: "run a default build in module directories if they have not been pre-built"},
 		cli.BoolFlag{Name: "f, force", Usage: buildForceUsage},
 		cli.BoolFlag{Name: "debug", Usage: debugUsage},
 	}
 
 	app.Commands = []cli.Command{
+		{
+			Name:   "init",
+			Usage:  "Scans your environment for code module entry points and writes to config",
+			Action: initCmd,
+			Flags: []cli.Flag{
+				cli.BoolFlag{Name: "f, force", Usage: "rescan and overwrite modules in config even if they exist"},
+				cli.BoolFlag{Name: "include-all", Usage: "include suspicious modules (`docs`, `test` or `example` in name)"},
+				cli.BoolFlag{Name: "skip-write", Usage: "do not write config if successful"},
```

Comment:
Why would I ever want this? I think a more sensible flag would be `--output`, which prints the configuration to `stdout` and then exits.

Reply:
Also if the user doesn't want to reformat stuff

Reply:
I don't understand. This is a flag on `fossa init`, a command which does nothing but generate an automatic configuration. This flag then throws away that configuration, causing `fossa init` to do nothing. Is that intended?